lisa/operating_system.py

```diff
@@ -504,26 +500,40 @@ def _package_exists(self, package: str, signed: bool = True) -> bool:
         return False
 
     def _get_information(self) -> OsInformation:
-        cmd_result = self._node.execute(
-            cmd="lsb_release -a", shell=True, no_error_log=True
-        )
+        # try to set version info from /etc/os-release.
+        cat = self._node.tools[Cat]
+        cmd_result = cat.run("/etc/os-release")
         cmd_result.assert_exit_code(message="error on get os information")
-        assert cmd_result.stdout, "not found os information from 'lsb_release -a'"
 
+        vendor: str = ""
+        release: str = ""
```

Comment:
I don't think we need to initialize these to empty string. They are already initialized to empty string in the OsInformation object.